net-mail/checkpassword-pam/checkpassword-pam-0.99-r1.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+IUSE=""
```

Comment:
Drop it, if empty. Also, if set, IUSE belongs under ```KEYWORDS``` ;-)

Reply:
done